tasks/release.py

```diff
@@ -40,6 +40,9 @@
 # Regex matching rc version tag format like 7.50.0-rc.1
 RC_VERSION_RE = re.compile(r'\d+[.]\d+[.]\d+-rc\.\d+')
 
+# Regex matching minor release rc version tag (no patch version) like 7.50.0-rc.1, but not 7.50.1-rc.1
```

Comment:
```suggestion
# Regex matching minor release rc version tag like x.y.0-rc.1 (semver PATCH == 0), but not x.y.1-rc.1 (semver PATCH > 0)
```
If I understand correctly a `minor release rc version`. I just want to highlight the difference with [semver](https://semver.org/) definitions :)